File: _array_1/Backjoon4344/Backjoon4344.java
package _array_1.Backjoon4344;

public class Backjoon4344 {
    public static void main(String args[]){
        int[] n = {5,50,50,70,80,100};
        int[] Nn = {0,0,0,0,0};
        double sum = 0.0, over = 0.0;
        for(int i = 1; i < n[0]; i++){
            sum += n[i];
            Nn[i-1] = n[i]; 
        }
        for(int i = 1; i < n[0]; i++){
            if(sum / n[0] < n[i])
                over += 1.0; 
        }
        System.out.print(over/n[0]*100 + "%");
    }
    
}

File: Baekjoon/Backjoon4344/Backjoon4344.py
n = [5,50,50,70,80,100]
Nn = list()
sum = 0.0
over = 0.0
for i in range(1,n[0]):
    sum += n[i]
    Nn.append(n[i])

for i in range(1,n[0]):
    if(sum / n[0] < n[i]):
        over += 1.0; 

print(over/n[0]*100,"%",sep="")